docs/Release_notes.md

```diff
@@ -1,5 +1,20 @@
 V2X-Hub Release Notes
 ----------------------------
+Version 7.2, released April 12th, 2022
+--------------------------------------------------------
+
+**Summary:**
+V2X Hub release 7.2 includes added functionality for the CARMACloud Plugin to support:
+- Receiving vehicle TCM acknowledgement messages and displaying appropriate positive/negative/no acknowledgement messages on V2X-Hub web UI.
```

Comment:
We need to include the issues for these enhancements below.

Reply:
Included new created issues #348 #349 #350 .